meta-genivi-dev/meta-ivi/recipes-extended/node-state-manager/node-state-manager_2.0.0.bbappend

```diff
@@ -1,5 +1,22 @@
-EXTRA_OECONF_append=" --with-nsmc=NodeStateMachineTest "
+#
+# This bbappend will add the Simple Node State Machine to the
+# node-state-manager build.
+#
+NSMC="SimpleNodeStateMachine"
+
+EXTRA_OECONF_append=" --with-nsmc=${NSMC} "
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+SRCREV_${NSMC} = "b4de5dee4d70a223eef37bcbb5f8536560b0e243"
+
+#
+# The SRC_URI append will checkout the commit at the location
+# hard-coded/configured in the EXTRA_OECONF above.
+# Also patch the node-state-manager configure.ac to build the
+# simple-node-state-machine.
+#
```

Comment:
I would prefer to keep the comment as it explains some of the reasoning behind the design that might be slightly unclear otherwise that depends on how node-state-manager build works internally. I could try to clarify it though? Would this be ok with you?